fern/docs/text-gen-solution/rest-api.mdx

```diff
@@ -167,7 +173,8 @@ Parameters
      _ **content** *(string)*: The actual text content of the chat completion.  
      _ **function_call** _(object or null)_: An optional field that may contain information about a function call made within the message. It's usually `null` in standard responses.  
      _ **delta** *(object or null)*: An optional field that can contain additional metadata about the message, typically `null`.  
-     _ **finish_reason** _(string)_: The reason why the message generation was stopped, such as reaching the maximum length (`"length"`).
+     _ **finish_reason** _(string)_: The reason why the message generation was stopped, such as reaching the maximum length (`"length"`).  
+     _ **logprobs** _(object)_: An object representing the token, its log probability and the most probable tokens to this one.
```

Comment:
"the most probable tokens to this one" what does this mean?

Reply:
Yes, it will be included in the response if we set the top_logprobs > 1. For example, we ask: "Create a story about a cat". 
The response will be in the following format:
```
"choices": [
    {
      "index": 0,
      "message": {
        "role": "assistant",
        "content": " In a quaint, cobblestone town,",
        "tool_calls": null
      },
      "logprobs": {
        "content": [
          {
            "token": " In",
            "logprob": -0.7101921439170837,
            "bytes": null,
            "top_logprobs": [
              {
                "token": " In",
                "logprob": -0.7101921439170837,
                "bytes": null
              },
              {
                "token": " Once",
                "logprob": -1.9485827684402466,
                "bytes": null
              }
            ]
          },
          .... other tokens
```